src/Tribe/Tickets.php

```diff
@@ -4368,14 +4369,20 @@ public function maybe_redirect_to_attendees_registration_screen( $redirect = nul
 			$meta = tribe( 'tickets-plus.meta' );
 
 			$cart_has_meta = true;
+			$cart_has_iac  = true;
 
-			// If the method exists (latest ET+ version), run it.
+			// If the method exists in ET+, run it.
 			if ( method_exists( $meta, 'cart_has_meta' ) ) {
 				$cart_has_meta = $meta->cart_has_meta( $tickets_in_cart );
 			}
```

Comment:
This should already cover the case for IAC, we should not add a new logic for that. IAC is just another form of meta.